docs/pages/router/advanced/customizing-links.mdx

```diff
@@ -0,0 +1,153 @@
+---
+title: Customizing links
+description: Learn how to perform link redirection and working with 3rd-party deep-links
+---
+
+import { FileTree } from '~/ui/components/FileTree';
+import { APIBox } from '~/components/plugins/APIBox';
+
+Expo Router offers a adaptable solution for managing navigation within your Expo app. However, its URL handling philosophy may not align with every use case or integration with third-party services. There are two common scenarios where you may need to customize a link:
```

Comment:
```suggestion
Expo Router uses an extended version of web-standards to navigate through an application. However, native apps do not always conform to server-based routing which can lead to misalignment when integrating with third-party services, e.g. apps can be launched with arbitrary strings or intent objects instead of URLs. There are two common scenarios where you may need to customize a link:
```